### PR TITLE
Document uninstall and Traefik, Helm integration (references #86, #96, #97)

### PR DESCRIPTION
**Pull request by matejgera**
Adds information about uninstall script, CRD for Helm Charts and Traefik integration / disabling to README.md. For more please see the respective issues #86, #96, #97.

### CODE REVIEW

**Comment by L8D** [comment]
👍 perfect timing. I'm just about to start using k3s and without this I would have no idea where to start with my Helm Charts!

**Comment by ibuildthecloud** [comment]
Wow, thanks!

File: README.md [hunk 1]
```diff
@@ -101,6 +101,21 @@ Auto-deploying manifests
...
+metadata:
+  name: traefik
+  namespace: kube-system
+spec:
+  chart: stable/traefik
+  set:
+    rbac.enabled: "true"
+    ssl.enabled: "true"
+```
+
```
Comment (ibuildthecloud) [inline README.md]: Do you want to put a reference to https://github.com/rancher/k3s/blob/master/types/apis/k3s.cattle.io/v1/types.go here. The type HelmChartSpec has the full reference of all fields that can be set in the Custom Resource.

**Comment by dpen2000** [comment]
@L8D You might encounter this issue currently: https://github.com/rancher/k3s/issues/151 (Can't remember whether it applies to Helm chart manifests as well offhand)

**Pull request merged**